packages/grid/src/components/column/Column.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { ColumnProps } from './Column.types';
+
+import styles from '../../styles/column.module.css';
+
+const breakpoints = ['sm', 'md', 'lg', 'xl'];
+
+function Column({
+  element: Wrapper = 'div',
+  children,
+  widths,
+  className,
+  offset,
+  order,
+}: ColumnProps): JSX.Element {
+  const widthClasses = widths.map(
+    (width, index) => styles[`${breakpoints[index]}-${width}`],
+  );
+
+  const classNames = cn(
+    styles.column,
+    ...widthClasses,
+    className,
+    offset && styles[`offset-${offset}`],
+    order && styles[`order-${order}`],
```

Comment:
try, should be works

```
cn({
  [styles[`offset-${offset}`]]: !!offset,
  [styles[`order-${order}`]]: order
}) 

```

Reply:
Decided to set order and offset according to breakpoint so they would be arrays instead, for example:
```
<Column offset={[3]} order={[1,2]}>
```